.github/workflows/ci_benchmarks.yml

```diff
@@ -47,14 +44,6 @@ jobs:
           echo "TEXT<<EOF" >> $GITHUB_OUTPUT
           asv continuous $ASV_OPTIONS ${BASE_SHA} ${GITHUB_SHA} 2>&1 | tee -a $GITHUB_OUTPUT
           echo "EOF" >> $GITHUB_OUTPUT
-      - name: Excerpt the summary
-        id: summary
-        run: |
-          echo "TEXT<<EOF" >> $GITHUB_OUTPUT
-          echo "${{ steps.asv.outputs.TEXT }}" | egrep "\| *Change *\| *Before *\[|BENCHMARKS NOT SIGNIFICANTLY CHANGED." -A 1000 >> $GITHUB_OUTPUT
-          echo "EOF" >> $GITHUB_OUTPUT
       - name: Post the summary
-        uses: thollander/actions-comment-pull-request@v3
-        with:
-          message: ${{ steps.summary.outputs.TEXT }}
-          comment-tag: asv_summary
+        run: |
```

Comment:
Could we use a separate invocation of [`asv compare`](https://asv.readthedocs.io/en/latest/commands.html#asv-compare) to get the results?

Reply:
From a quick poke about locally, the answer seems to be mostly yes. The output isn't exactly the same though.